recipes/mplt/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "mplt" %}
+{% set version = "2.0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "2e755db3e8efa9c4dc7728f063458280738d8a4c28e455966c77151173ac791b"
```

Could you please remove the quotes from the ``sha256`` ?
```suggestion
  sha256: 2e755db3e8efa9c4dc7728f063458280738d8a4c28e455966c77151173ac791b
```